public/locales/es/modules/usenet.json

```diff
@@ -1,7 +1,7 @@
 {
   "descriptor": {
-    "name": "Sabnzbd",
-    "description": "Te permite ver tu cola e historial de Sabnzbd, pausar y resumir descargas"
```

Comment:
I don't know what languages you're speaking, but don't **need** to translate, because we have an awesome translation community that does this. But since you already did this, we can just leave it like that.

More info: https://homarr.dev/docs/community/translations

Reply:
Oh ok! Good to know. I just used Google Translate. :)